shell/browser/api/electron_api_cookies.cc

```diff
@@ -171,6 +190,22 @@ std::string InclusionStatusToString(
   return "Setting cookie failed";
 }
 
+net::CookieSameSite StringToCookieSameSite(const std::string* str_ptr) {
+  if (!str_ptr)
+    return net::CookieSameSite::NO_RESTRICTION;
+  std::string str = *str_ptr;
```

Comment:
Copying`str_ptr` is unnecessary. If you want `str` for the readability of saying `str` instead of `*str_ptr`, you could use a const reference:

```suggestion
  const std::string& str = *str_ptr;
```